<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1431</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonStart">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>20</y>
      <width>167</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>20</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>60</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3">
    <property name="geometry">
     <rect>
      <x>1310</x>
      <y>100</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSeeTime">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>60</y>
      <width>167</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>See time</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOutput">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>100</y>
      <width>167</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Output data in file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInput">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>180</y>
      <width>167</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Input data from file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonInputManually">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>140</y>
      <width>167</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Input data</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1431</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNEW">
   <property name="text">
    <string>NEW</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;new text&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
